### DISCUSSION TITLE
change validate password for sip extension

### DISCUSSION THREAD

**Discussion by answerwar** [mikopbx/Core#831] [General ( Обо всём )]
Hello, I want to remove or change the validate on the password for SIP, because the password we are currently using is only numbers. Where is the file for custom SIP passwords on the admin cabinet?

**Comment by boffart** [comment]
the task can be solved through customization

Open the employee's account for editing
Go to "**Advanced Settings**" → "**Advanced SIP Settings**"
Add the following lines:
```
[auth] 
password = 123
```

**Reply by Swedlund** [reply]
Any option to disable sip authentication completely for an extension, e.g no password?
